graphics/rendering/blender/README.md

```diff
@@ -13,36 +13,42 @@ In this example, we will install Blender in a [Ubuntu container](https://cloud.s
 
 <br>
 
-### To start, Make the working directory:
+## To start, Make the working directory:
 
 ```
 $ mkdir ~/blender
 $ cd ~/blender/
 ```
 
+We will run this example in the `~/blender/` directory, not necessary but its cleaner this way.
+
 <br>
 
-Then, pull the container from the library:
+### Then, pull the container from the library:
```

Comment:
change title by "Pull the container from library"

Reply:
:+1: